pandas/tests/io/test_html.py

```diff
@@ -3,13 +3,17 @@
 import glob
 import os
 import re
+import threading
 import warnings
 
+
+# imports needed for Python 3.x but will fail under Python 2.x
```

Comment:
`reload` is a built-in in Python2. There is no need to import it unless you are using Python3.